### PR TITLE
Fix alarm usages

### LINKED ISSUE #129: Alarm is never fired
After upgrading to gRPC 1.7.2, alarm is not fired anymore, which lead to hang forever in threaded application.

### PR DESCRIPTION
**Pull request by BusyJay**
close #129.

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: src/async/executor.rs [hunk 1]
```diff
@@ -36,7 +37,9 @@ impl Alarm {
             let ptr = Box::into_raw(tag);
             let timeout = GprTimespec::inf_future();
             let cq_ref = cq.borrow()?;
-            grpc_sys::grpc_alarm_create(cq_ref.as_ptr(), timeout, ptr as _)
```
Comment (overvenus) [inline src/async/executor.rs]: But why it compiles?

Comment (BusyJay) [reply src/async/executor.rs]: Why can't it compile?

Comment (overvenus) [reply src/async/executor.rs]: The signature has changed.

Comment (BusyJay) [reply src/async/executor.rs]: The signature of C/C++ code changes, not the API written in rust. Rust compiler can just compile rust code.

Comment (overvenus) [reply src/async/executor.rs]: So why does the rustc find the changed C API?

Comment (BusyJay) [reply src/async/executor.rs]: Because they share the same name. See also #130.

**Comment by BusyJay** [comment]
PTAL

**Pull request merged**